### PR TITLE
truncates text in clipboard

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|cms_ipad_clipboard|screen shot 2015-11-18 at 08 44 38|

### CODE REVIEW

File: cms/static/cms/sass/components/_clipboard.scss [hunk 1]
```diff
@@ -20,6 +20,7 @@
         cursor: move;
     }
     .cms-draggable .cms-dragitem {
+        overflow: hidden;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_clipboard.scss]: is there a need for overflow here?

Comment (lory87) [inline cms/static/cms/sass/components/_clipboard.scss]: yes to remove horizontal scrollbar

Comment (lory87) [inline cms/static/cms/sass/components/_clipboard.scss]: @vxsx was wrong with this solution it's not necessary. removed

**Comment by vxsx** [comment]
:+1:

**Pull request merged**